konig-schemagen-maven-plugin/src/main/java/io/konig/maven/KonigSchemagenMojo.java

```diff
@@ -346,6 +350,13 @@ private void generateEntityStructure(File webappDir) throws KonigException, IOEx
 		EntityStructureWorker worker = new EntityStructureWorker(nsManager, shapeManager, baseDir);
 		worker.run();
 	}
+	
+	private void generateBigQueryFunction(File webappDir) throws KonigException, IOException {
+		File baseDir = new File(webappDir, "WEB-INF/classes/Fucntions");
```

Comment:
I have a few observations and suggestions.

* You have a typo in the name "Functions".
* The name "Functions" seems too generic.  I suggest that we use "GoogleAnalyticsExport" as the name of the folder.
* As a best practice, you should not write anything directly to "WEB-INF/classes/".  Instead, the best practice is to put resources into "src/main/resources".  They will then get copied into "WEB-INF/classes" via the maven build process.  In short, you should use the following...
```
File baseDir = new File(mavenProject.getBasedir(), "src/main/resources/GoogleAnalyticsExport");
```